mitodl/teachersportal
# Header nav username

#### What are the relevant tickets?

Refs #167 
#### What's this PR do?

Makes the header look a lot more like the mockup.
#### Where should the reviewer start?

`Navigation.js` for the FE stuff
`Webpack.py` for the BE stuff.
#### How should this be manually tested?

It should look nice and when you log in, your full name should appear in the top right. It should stay there if you refresh.
#### Any background context you want to provide?

Based off of #328. Compare here: https://github.com/mitodl/teachersportal/compare/course-list-and-homepage...header-nav-username
#### Screenshots (if appropriate)

![better-header-photo](https://cloud.githubusercontent.com/assets/3853/12827232/a82403b8-cb4c-11e5-86b9-649dfece4d4e.png)
#### What gif best describes this PR or how it makes you feel?

![](http://s.mlkshk.com/r/17GKE.jpg)
